### PR TITLE
Implement a skeletal version of FridgeFriend that starts by greeting …

### LINKED ISSUE #13: Implement a skeletal version of FridgeFriend
Implement a skeletal version of FridgeFriend that starts by greeting the user, simply echos commands entered by the user, and exits when the user types bye.

### PR DESCRIPTION
**Pull request by Leeyp**
…the user, simply echos commands entered by the user, and exits when the user types bye.

### CODE REVIEW

**Review by Vinci-Hu — COMMENTED** [review]
Shouldn't we be using packages and separate classes for parsing commands already?

File: src/main/java/seedu/duke/Duke.java [hunk 1]
```diff
@@ -3,19 +3,20 @@
 import java.util.Scanner;
 
 public class Duke {
-    /**
-     * Main entry-point for the java.duke.Duke application.
-     */
     public static void main(String[] args) {
```
Comment (kwokyto) [inline src/main/java/seedu/duke/Duke.java:6]: I think it would be better if you refactor the functions out of 'main'.

**Comment by kwokyto** [comment]
> Shouldn't we be using packages and separate classes for parsing commands already?

I think we can use packages, but not classes for parsing as of now. We will handle parsing in another commit.

File: src/main/java/seedu/duke/Duke.java (hunk 1 above)
Comment (kwokyto) [inline src/main/java/seedu/duke/Duke.java:6]: Would it better to refactor all the key functions out of the 'main' method?

File: src/main/java/seedu/duke/Duke.java [hunk 2]
```diff
@@ -3,19 +3,20 @@
...
-        System.out.println("What is your name?");
 
+        System.out.println("Hello! I'm FridgeFriend!");
+        System.out.println("What can I do for you?\n");
+
+        String input;
         Scanner in = new Scanner(System.in);
-        System.out.println("Hello " + in.nextLine());
+        input = in.nextLine();
+        while (!input.equals("bye")){
```
Comment (kwokyto) [inline src/main/java/seedu/duke/Duke.java]: Would it be better to use a switch statement here? This can help accommodate future addition of commands.

**Review by kwokyto — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/Duke.java (hunk 1 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java:6]: Alright

File: src/main/java/seedu/duke/Duke.java (hunk 2 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java]: Alright

**Comment by Leeyp** [comment]
Resolved comments in second commit.

**Comment by joohwan58** [comment]
The switch statement is missing the `break;` for the cases

File: src/main/java/seedu/duke/Duke.java [hunk 3]
```diff
@@ -5,18 +5,34 @@
...
-        while (!input.equals("bye")){
-            System.out.println(input + "\n");
-            input = in.nextLine();
+
+        boolean isBye = false;
+        while (!isBye) {
+            switch (input) {
+            case "bye":
+                isBye = true;
+                byeMessage();
```
Comment (joohwan58) [inline src/main/java/seedu/duke/Duke.java]: missing `break;`

File: src/main/java/seedu/duke/Duke.java [hunk 4]
```diff
@@ -5,18 +5,34 @@
...
+            default:
+                System.out.println(input + "\n");
+                input = in.nextLine();
```
Comment (joohwan58) [inline src/main/java/seedu/duke/Duke.java:23]: missing `break;`

File: src/main/java/seedu/duke/Duke.java [hunk 5]
```diff
@@ -3,19 +3,36 @@
...
+            }
+        }
+
+
+
+    }
+
+    private static void byeMessage() {
```
Comment (Vinci-Hu) [inline src/main/java/seedu/duke/Duke.java]: Function name should be a verb, like printByeMessage

File: src/main/java/seedu/duke/Duke.java [hunk 6]
```diff
@@ -3,19 +3,36 @@
...
+        System.out.println("Bye. Hope to see you again soon!\n");
+    }
+
+    private static void welcomeMessage() {
```
Comment (Vinci-Hu) [inline src/main/java/seedu/duke/Duke.java]: Same as above

File: src/main/java/seedu/duke/Duke.java [hunk 7]
```diff
@@ -5,18 +5,34 @@
...
         String input;
         Scanner in = new Scanner(System.in);
         input = in.nextLine();
-        while (!input.equals("bye")){
-            System.out.println(input + "\n");
-            input = in.nextLine();
+
+        boolean isBye = false;
+        while (!isBye) {
+            switch (input) {
```
Comment (SimJJ96) [inline src/main/java/seedu/duke/Duke.java]: Would it be better to consider the input command as a lower case to handle the upper case command of bye?

File: src/main/java/seedu/duke/Duke.java (hunk 3 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java]: Resolved!

File: src/main/java/seedu/duke/Duke.java (hunk 4 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java:23]: Resolved!

File: src/main/java/seedu/duke/Duke.java (hunk 5 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java]: Resolved!

File: src/main/java/seedu/duke/Duke.java (hunk 6 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java]: Resolved!

File: src/main/java/seedu/duke/Duke.java (hunk 7 above)
Comment (Leeyp) [reply src/main/java/seedu/duke/Duke.java]: resolved!

**Comment by kwokyto** [comment]
LGTM

**Review by kwokyto — APPROVED** [review]

**Comment by Vinci-Hu** [comment]
LGTM

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Review by Vinci-Hu — APPROVED** [review]

**Pull request merged**